test/jquery.dbEditorCombo.test.html

```diff
@@ -0,0 +1,5 @@
+<div class="container" style="position:relative">
+  <div class="editor"
+       style="margin: 50px; border: 1px solid grey;"
+       >Fruit...</div>
```

Comment:
This is a non-standard html formatting technique.
Is there an agreed standard formatter with indentation , line continuation etc ?

Reply:
Looking at https://developers.google.com/style/html-formatting and https://google.github.io/styleguide/htmlcssguide.html I'd say this is pretty standard, is there a specific change you'd prefer?